python/lsst/meas/base/tests.py

```diff
@@ -485,7 +498,7 @@ def addSource(self, instFlux, centroid, shape=None, setPeakSignificance=True):
         image = self.drawGaussian(self.exposure.getBBox(), instFlux,
                                   lsst.afw.geom.Ellipse(fullShape, centroid))
         # Generate a footprint for this source
-        self._installFootprint(record, image, setPeakSignificance)
+        self._installFootprint(record, image, setPeakSignificance, negative=negative)
```

Comment:
negative=negative? Just checking, is this what you intended?

Reply:
Yes: the user calls `addSource` and supplies `negative` if desired, this just passes it through.